packages/paste-core/utilities/grid/__tests__/grid.test.tsx

```diff
@@ -0,0 +1,588 @@
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={5}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={5}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={2}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should render a 6 span, 6 span column grid', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={6}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={6}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should render a 8 span, 4 span column grid', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={8}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={4}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should render a 9 span, 3 span column grid', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={9}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={3}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should render a 10 span, 2 span column grid', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column span={10}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column span={10}>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should render a 11 span, 1 span column grid', (): void => {
```

Comment:
We should test:
- a `5span 3span` column grid (sum <12)
- a `11span 2span` column grid (sum >12)

Reply:
I'll add these tests. They don't return any errors, but the column math gets wonky. We should consider adding a console warning saying something about `span` totally out to 12. We are adding information to the spec doc about the 12 column support.